### PR TITLE
WasikHM1

### PR DESCRIPTION
**Pull request by SarahWasik**
I wrote my Markdown file locally in GitHub. I also copy pasted it into R-Studio to demonstrate to myself my code would work on both, but it seemed easier to do Markdown locally for now. The difficulties with this assignment involved the spacing, mostly. However, if you preview your work enough you can figure it out. I also had some issues with the picture-- I thought you could use a picture locally but you need to upload the photo to your Github account. It would be nice if Github had spell check because I am terrible at spelling. I also have no idea how to submit assignments so hopefully this works..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # hw01
```
Comment (jenssl) [inline README.md:1]: Reviewed by Sarah J

Comment (jenssl) [reply README.md:1]: Sarah- You used many of the elements suggested in the rubric including headers, lists, links, and pictures and did successfully upload your markdown file (I read your comments). In addition, your document is clean and easy to read. I noticed one small typo when you introduce your picture (perhaps missing "of"); however, this does not interfere with any of the coding. I might suggest finding a way to scale your picture so it's not so big. This is something I encountered when uploading my own photo, and unfortunately, I have not yet found a solution. Additionally, I like how you included in-text links. I didn't know you could attach links this way. Your approach is creative and makes your document easy to navigate. Overall, nice job!

**Review by jenssl — COMMENTED** [review]
Again, good job putting this together.

**Comment by cysakamoto** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Hi Sarah, 

Great use of markdown elements in your page, also using links and adding a picture. Glad to hear you've been discovering how to work your way around issues that might come up. Watch for typos in your codes, as the smallest mistake could give you errors. 

Cristina

**Pull request closed without merging**